controller/internal/routes/edge_router_api_model.go

```diff
@@ -174,17 +150,176 @@ func MapEdgeRouterToApiList(ae *env.AppEnv, i *model.EdgeRouter) (*EdgeRouterApi
 		}
 
 		if len(enrollments) != 1 {
-			return nil, fmt.Errorf("expected enrollment not found for unverified edge router %s", i.Id)
+			return nil, fmt.Errorf("expected enrollment not found for unverified edge router %s", router.Id)
 		}
 		enrollment := enrollments[0]
 
-		ret.EnrollmentExpiresAt = enrollment.ExpiresAt
-		ret.EnrollmentCreatedAt = &enrollment.CreatedAt
+		createdAt := strfmt.DateTime(enrollment.CreatedAt)
+		expiresAt := strfmt.DateTime(*enrollment.ExpiresAt)
+
+		ret.EnrollmentExpiresAt = &expiresAt
+		ret.EnrollmentCreatedAt = &createdAt
 		ret.EnrollmentJwt = &enrollment.Jwt
 		ret.EnrollmentToken = &enrollment.Token
 	}
 
-	ret.PopulateLinks()
-
 	return ret, nil
 }
+
+//
+//func NewEdgeRouterEntityRef(entity *model.EdgeRouter) *EntityApiRef {
```

Comment:
do you want to keep all this commented out code?

Reply:
Nope, was using them as reference. Forgot to remove this one.